src/Form/UserRelatedDataType.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Form;
+
+use App\DTO\RelatedLinkDTO;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\DataMapperInterface;
+use Symfony\Component\Form\Exception\UnexpectedTypeException;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class UserRelatedDataType extends AbstractType implements DataMapperInterface
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('label', TextType::class)
+            ->add('link', UrlType::class)
```

Comment:
The url type force this value to be a url. Even if I insert something normal, it still comes out as a url. In general I think this variable/field should be called `value` instead

Reply:
modified link by value